### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by NivLipetz**
add FAQ page to documentary

### CODE REVIEW

File: docs/devguide/docs/tests.md [hunk 1]
```diff
@@ -43,12 +43,12 @@ will randomly store `45697038-...-ed7a4a20e014` or `f02392e8-...-8a9a1ae5c47a` i
 
 ## Built-in Functions
 
-Predator supports some generic functions out of the box and they can be used to generate random data
+Predator supports some generic functions out of the box and they can be used to generate random data.
```
Comment (enudler) [inline docs/devguide/docs/tests.md]: lets add that everything generated here is a string

File: docs/devguide/docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Installation
+
+### <b>Predator is running, how do I access the UI?</b>
+
+The UI is accessible at http://$MACHINE_IP/ui where `$MACHINE_IP` is your local network address you used to install Predator.
+                        
+## Tests
+
+### <b>I run a test successfully but no report is created for the test</b>
```
Comment (enudler) [inline docs/devguide/docs/faq.md]: why not write what we wrote in the installation part?

File: docs/devguide/docs/tests.md (hunk 1 above)
Comment (NivLipetz) [reply docs/devguide/docs/tests.md]: but it's a bug. I think we should solve it. No?

File: docs/devguide/docs/faq.md (hunk 2 above)
Comment (NivLipetz) [reply docs/devguide/docs/faq.md]: good point

Comment (NivLipetz) [reply docs/devguide/docs/faq.md]: actually, i think it's too detailed there and this is a good sum up of the actual problem + solution. In the end it's the same solution but worded a tiny bit differently

File: docs/devguide/docs/tests.md (hunk 1 above)
Comment (enudler) [reply docs/devguide/docs/tests.md]: i have no idea... but at least lets say the truth for now

File: docs/devguide/docs/faq.md (hunk 2 above)
Comment (enudler) [reply docs/devguide/docs/faq.md]: ok so maybe add link to the installation docker anchor

**Pull request merged**